include/pdflib.h

```diff
@@ -681,6 +681,22 @@ PDF_info_textline(PDF *p, const char *text, int len, const char *keyword,
 PDFLIB_API void PDFLIB_CALL
 PDF_initgraphics(PDF *p);
 
+/* Open a disk-based or virtual vector graphics file subject to various options. */
+PDFLIB_API int PDFLIB_CALL
+PDF_load_graphics(PDF *p, const char *type, const char *filename, int len, const char *optlist);
+
+/* Close vector graphics. */
+PDFLIB_API void PDFLIB_CALL
+PDF_close_graphics(PDF *p, int graphics);
+
+/* Place vector graphics on a content stream, subject to various options. */
+PDFLIB_API void PDFLIB_CALL
+PDF_fit_graphics(PDF *p, int graphics, double x, double y, const char *optlist);
+
+/* Format vector graphics and query metrics and other properties. */
+PDFLIB_API double PDFLIB_CALL
+PDF_info_graphics(PDF *p, int graphics, const char *keyword, const char *optlist);
+
```

Comment:
Could these method definitions be in alphabetical order, among all the other method definitions?